gtfs/spec/en/reference.md

```diff
@@ -379,20 +379,25 @@ Shapes describe the physical path that a vehicle takes, and are defined in the f
+|  trip_id | **Required** | The **trip_id** contains an ID that identifies a trip on which the specified headway of service applies. Trip IDs are referenced from the [trips.txt](#tripstxt) file. |
+|  start_time | **Required** | The **start_time** field specifies the time at which the first vehicle departs from the first stop of the trip with the specified headway. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service day. For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. E.g. 25:35:00. |
+|  end_time | **Required** | The **end_time** field indicates the time at which service changes to a different headway (or ceases) at the first stop in the trip. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service day. For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. E.g. 25:35:00. |
 |  headway_secs | **Required** | The **headway_secs** field indicates the time between departures from the same stop (headway) for this trip type, during the time interval specified by **start_time** and **end_time**. The headway value must be entered in seconds. |
 |   |  | Periods in which headways are defined (the rows in frequencies.txt) shouldn't overlap for the same trip, since it's hard to determine what should be inferred from two overlapping headways. However, a headway period may begin at the exact same time that another one ends, for instance: |
 |   |  | `A, 05:00:00, 07:00:00, 600` |
 |   |  | `B, 07:00:00, 12:00:00, 1200` |
-|  exact_times | Optional | The **exact_times** field determines if frequency-based trips should be exactly scheduled based on the specified headway information. Valid values for this field are: |
-|   |  |* **0** or **(empty)** - Frequency-based trips are not exactly scheduled. This is the default behavior. |
-|   |  |* **1** - Frequency-based trips are exactly scheduled. For a frequencies.txt row, trips are scheduled starting with trip_start_time = start_time + x * headway_secs for all x in (0, 1, 2, ...) where trip_start_time < end_time. |
+|  exact_times | Optional | The **exact_times** field determines if trips represent frequency-based service or scheduled-based service using on the specified headway information. Valid values for this field are: |
+|   |  |* **0** or **(empty)** - Trips are frequency-based. Operators make their best effort to serve stops periodically based on a headway, but there is no fixed schedule.  This is the default behavior. |
```

Comment:
@barbeau What do you think about adding an extra note in here to specify expectations around a UI? Or is this superfluous? Proposed additions are in **_bold italic_**.

 |  Field Name | Required | Details |
  |  ------ | ------ | ------ |
|  exact_times | Optional | The **exact_times** field determines if trips represent frequency-based service or scheduled-based service using on the specified headway information. Valid values for this field are: |
 |   |  |* **0** or **(empty)** - Trips are frequency-based. Operators make their best effort to serve stops periodically based on a headway, but there is no fixed schedule. **_A user interface should therefore not show scheduled pickup and drop-off times._** This is the default behavior. |
|   |  |* **1** - Trips are scheduled. For a frequencies.txt row, trips are scheduled starting with trip_start_time = start_time + x * headway_secs for all x in (0, 1, 2, ...) where trip_start_time < end_time. **_A user interface may show scheduled pickup and drop-off times._** |